### ISSUE TITLE
Create new "release" of ningen, so gtkcord3 can build

### ISSUE THREAD

**Issue by Titaniumtown** [diamondburned/ningen#4]
I made a PR (diamondburned/gtkcord3/pull/94) on gtkcord3 (you can read more about it there) but gtkcord3 fails to build because a new release of ningen hasn't been created, so the folder "handlerrepo" isn't found, and the build fails. Hope this issue can be resolved!

**Comment by Titaniumtown** [comment]
Nevermind, this can be fixed on gtkcord3's side.

**Comment by PaulDGillis** [comment]
@diamondburned Would it be possible to get an updated release? While gtkcord3 lists ningen as v0.1.1-{something or other} it always falls back to 0.1.0 for me. I think this also may be why the latest commit on gtkcord3 master failed its [build check](https://builds.sr.ht/~diamondburned/job/262599). 

Currently I have a local copy of arikawa and ningen that I'm developing with where I've fixed the compilation. I've also updated arikawa to 1.3.2 in my local copy of gtkcord3 and plan to submit a pr. I just wanted to ask if we could get an updated ningen to go along with the update to arikawa 1.3 for the pr. 

Arikawa has had about 20 releases since gtkcord3(AK v0.9.6) last updated it, and about 21 since ningen v0.1.0(AK v0.9.5). The pr would keep gtkcord from falling too far behind, since I was hoping to implement status in gtkcord after fixing an [issue I created on arikawa](https://github.com/diamondburned/arikawa/issues/140).
